### PR TITLE
Password_Generator_App.py

### PR DESCRIPTION
**Pull request by Raghavarora27**
I  have made this Password generator app using python and tkinter....

### CODE REVIEW

File: Python/Password_generator_App.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            s.extend(list(s4))
+            random.shuffle(s)
+            pas = "".join(s[0:passlen])
+            lbl = tk.Label(root,text=pas,font=("times new roman",15,"bold")).place(x=50,y=170,width=250)
+    except:
+        pass
+def finish():
+    exit()
+
+labl = tk.Label(root,text="Enter the Length of the Password :-",font=("times new roman",15,"bold")).place(x=10,y=20)
```
Comment (kaustubhgupta) [inline Python/Password_generator_App.py:40]: Just refactor these lines, all in the same line doesn't look nice. I am looking for something similar to this:
```
tk.label(root,
       text = "fdf"
       font = 'df'   
```
Like this

File: Python/Password_generator_App.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Length_of_password = tk.Entry(root,font=('Ubuntu',16),textvariable=x).place(x=10,y=70,height=30,width=330)
+but = tk.Button(root,text="Generate the Password",fg="white",command=generatepass,font=("times new roman",15,"bold"),bg="black").place(x=10,y=130,height=30,width=330)
+but1 = tk.Button(root,text="QUIT",fg="white",command=finish,font=("times new roman",15,"bold"),bg="red").place(x=10,y=220,height=30,width=330)
+
+root.mainloop()
```
Comment (kaustubhgupta) [inline Python/Password_generator_App.py:45]: - Add the working GIF of this password manager and put both the files (code + GIF) in a separate folder under the Python directory. 
- Make changes in the language readme file with your entry as "password manager -2"

**Review by kaustubhgupta — APPROVED** [review]

File: Python/Password_generator_App.py (hunk 1 above)
Comment (Raghavarora27) [reply Python/Password_generator_App.py:40]: ok i will do it!!!

File: Python/Password_generator_App.py (hunk 2 above)
Comment (Raghavarora27) [reply Python/Password_generator_App.py:45]: ok!!

Comment (Raghavarora27) [reply Python/Password_generator_App.py:45]: brother i am facing problem in pushing to github...Its saying permission denied..Can u help me i am new to github..

Comment (kaustubhgupta) [reply Python/Password_generator_App.py:45]: you need to push changes to your forked repo and not my repo. The changes you will make in the forked repo will be reflected in my repo

**Pull request merged**